src/endpoint/s3/s3_rest.js

```diff
@@ -348,7 +346,7 @@ async function authorize_request_iam_policy(req) {
     const resource_arn = _get_arn_from_req_path(req);
     const method = _get_method_from_req(req);
     const iam_policies = account.iam_user_policies || [];
-    if (iam_policies.length === 0) return;
+    if (iam_policies.length === 0 && req.object_sdk.nsfs_config_root) return; // We do not have IAM policies in NC yet
```

Comment:
This PR more like disabling iam_policy authorize check for NC?

Reply:
Yes, we do not have an IAM user policy in NC, so the default behavior in NC is that the user can perform S3 operations after it has access keys.
In this PR, we change the default behavior in containerized deployments that must have an IAM user policy for S3 operations.